src/vtktools.jl

```diff
@@ -263,3 +410,94 @@ function calc_vtk_points_cells(::Val{3}, coordinates::AbstractMatrix{Float64},
   return vtk_points, vtk_cells
 end
 
+
+# Convert coordinates and level information to a list of points and VTK cells (2D version)
```

Comment:
This method and the next one are only used for the curved stuff, right? Could we please have a comment that mentions this?